h/static/scripts/annotator/anchoring/pdf.coffee

```diff
@@ -18,6 +18,8 @@ getSiblingIndex = (node) ->
 
 
 getNodeTextLayer = (node) ->
+  if node is null
```

Comment:
Perhaps I'm not understanding the source of the bug here, but why is this check required? Couldn't we just not call `getNodeTextLayer` with a `null` argument?